cmd/influxd/run.go

```diff
@@ -89,6 +90,35 @@ func execRun(args []string) {
 			go func() { log.Fatal(http.ListenAndServe(config.ApiHTTPListenAddr(), sh)) }()
 		}
 		log.Printf("DataNode#%d running on %s", s.ID(), config.ApiHTTPListenAddr())
+
+		// Spin up any Graphite servers
+		for _, c := range config.Graphites {
+			if !c.Enabled {
+				continue
+			}
+
+			// Configure Graphite parsing.
+			parser := graphite.NewParser()
+			parser.Separator = c.NameSeparator
```

Comment:
What if this is blank?  It defaults to `.` when you create a new one, but if the config doesn't specify it, then you have an empty string as a separator.  I could be missing the check for this later on.